### ISSUE TITLE
Update ReadMes and Markdown Info

### ISSUE THREAD

**Issue by MasterCash** [JacobFischer/Cadre#19]
The current ReadMes and Markdown Info seem to be out of date for most systems or are inconsistent.

**Comment by JacobFischer** [comment]
Yes that is probably correct. It can't hurt to take a second look at all of them

**Comment by JacobFischer** [comment]
These are still out of date and need to be addressed. Users just reported errors trying to follow their old steps
